src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,10 +4,15 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
+        logger.info("UserId was called. Params: userId = {}", userId);
```

Comment:
"UserId was called.@ is incorrect message and not informative, use message like login method